src/django/api/views.py

```diff
@@ -1,17 +1,36 @@
+from django.db import transaction
 from django.core.exceptions import PermissionDenied
 from rest_framework.generics import CreateAPIView
 from rest_framework.decorators import api_view, permission_classes
 from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
 from rest_auth.views import LoginView, LogoutView
 
+from api.models import Organization, User
 from api.serializers import UserSerializer
 
 
 @permission_classes((AllowAny,))
 class SubmitNewUserForm(CreateAPIView):
     serializer_class = UserSerializer
 
+    @transaction.atomic
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+
+        if serializer.is_valid(raise_exception=True):
+            serializer.save()
+            pk = serializer.data['id']
+            user = User.objects.get(pk=pk)
+
+            Organization.objects.create(
+                admin=user,
+                name=user.email,
```

Comment:
The Organization name should come from this field on the registration form.

<img width="1125" alt="screen shot 2019-01-18 at 9 27 04 am" src="https://user-images.githubusercontent.com/17363/51399482-4f40a800-1b03-11e9-8df3-be2b3708e174.png">

Reply:
Addressed in 
933b9b2